src/views/Farm/components/Stake.tsx

```diff
@@ -73,7 +74,7 @@ const Stake: React.FC<StakeProps> = ({ lpContract, pid, tokenName }) => {
       <CardContent>
         <StyledCardContentInner>
           <StyledCardHeader>
-            <CardIcon>👨🏻‍🍳</CardIcon>
+            <CardIcon><img src={BeeIcon} /></CardIcon>
```

Comment:
What you think of removing the `CardIcon` and use the image alone.

Reply:
Removed the inlay and the background color, but the CardIcon component has some margin and css we want to keep.